src/redux/books/booksSlice.js

```diff
@@ -1,53 +1,38 @@
 const ADD_BOOK = 'bookstore/ADD-BOOK';
-const REMOVE_BOOK = 'bookstore/REMOVE-BOOK';
+const URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/HQ4dyPd09pUjSNKdipg5/books';
 
-const books = [
-  {
-    itemId: 'item1',
-    comments: [],
-    completed: 64,
-    category: 'Fiction',
-    title: 'The Great Gatsby',
-    author: 'John Smith',
-    chapters: 28,
-    presentChapter: { chapter: 17, chapterTitle: '' },
-  },
-  {
-    itemId: 'item2',
-    comments: [],
-    completed: 8,
-    category: 'Fiction',
-    title: 'Anna Karenina',
-    author: 'Leo Tolstoy',
-    chapters: 50,
-    presentChapter: { chapter: 3, chapterTitle: '' },
-  },
-  {
-    itemId: 'item3',
-    comments: [],
-    completed: 0,
-    category: 'Nonfiction',
-    title: 'The Selfish Gene',
-    author: 'Richard Dawkins',
-    chapters: 15,
-    presentChapter: { chapter: 1, chapterTitle: '' },
-  },
-];
-
-export const addBook = (book) => ({
+export const getBook = (book) => ({
   type: ADD_BOOK, book,
 });
 
-export const removeBook = (itemId) => ({
-  type: REMOVE_BOOK, itemId,
-});
+export const fetchBooks = () => (dispatch) => {
+  fetch(URL)
+    .then((response) => response.json())
+    .then((data) => dispatch(getBook(data)));
+};
```

Comment:
- [ ] as required in the project requirement, Kindly use `createAsyncThunk` to make API calls. Check out this [link](https://redux-toolkit.js.org/api/createAsyncThunk) for more info.